### ISSUE TITLE
Poor qualitative results

### ISSUE THREAD

**Issue by rsrinath14** [LiquidAmmonia/ColorNeRF#1]
Hi! Thanks for sharing the code. I was trying to train the "ninja" scene from LLFF with the instructions given in the README section, but the qualitative results that I'm getting are poor. 

Here are the commands that I used to train the scene:
For stage 1:
CUDA_VISIBLE_DEVICES=3,4 python train_color.py \
   --dataset_name llff  \
   --root_dir /data/test/ColorNeRF/data/LLFF_Dataset/nerf_llff_data/ninja \
   --N_importance 64 --img_wh 640 360 \
   --num_epochs 40 --batch_size 8192 \
   --optimizer adam --lr 5e-4 \
   --lr_scheduler steplr --decay_step 10 20 --decay_gamma 0.5 \
   --exp_name test_ninja \
   --loss_type color \
   --num_gpus 2 \
   --num_workers 32 \
   --not_use_patch \
   --normalize_illu \
   --local_run \
   --val_sanity_epoch 1
   
 For stage 2:
 CUDA_VISIBLE_DEVICES=3,4 python train_color.py \
   --dataset_name llff \
   --root_dir/data/srinath/ColorNeRF/data/LLFF_Dataset/nerf_llff_data/ninja \
   --weight_path /data/test/ColorNeRF/ckpts/test_ninja/last.ckpt \
   --N_importance 64 --img_wh 640 360 \
   --num_epochs 30 --batch_size 4096 \
   --optimizer adam --lr 5e-4 \
   --lr_scheduler steplr --decay_step 10 20 30 40 --decay_gamma 0.1 \
   --exp_name test_bike_stage_2 \
   --loss_type color \
   --num_gpus 2 \
   --num_workers 64 \
   --use_patch \
   --teacher_model 'ct2' \
   --train_stage 2 \
   --patch_sample_method central \
   --local_run \
   --use_color_hist \
   --use_color_class_loss \
   --not_color_hist_force_accept \
   --color_hist_thres 0.90 \
   --chunk 8192 \
   --use_tv_loss 
 
Results:
![ninja](https://github.com/LiquidAmmonia/ColorNeRF/assets/47494475/7ab67f54-04ff-4c8a-a779-403d3631b68a)

Please let me know if I'm missing out anything important. Thanks in advance!

**Comment by LiquidAmmonia** [comment]
Hi! Thank you for take interest in my work, the above results mainly attribute to poor grayscale nerf results (see the blurry geometry), for geometry, we basically use nerf_pl to construct the scene. Please change the hyper-parameters in stage 1 to get a better grayscale nerf

**Comment by rsrinath14** [comment]
Hi @LiquidAmmonia, thanks for your reply. Can you please provide the details of the hyper-parameters to get the results obtained in the paper? I tried the hyperparameters provided in the README section to test it on the flower scene in LLFF. The weren't great there too. 
Also can you please point at nerf_pl ?

**Comment by LiquidAmmonia** [comment]
nerf_pl at https://github.com/kwea123/nerf_pl;
I used their llff hyper-parameters for scenes like flower.

**Comment by rsrinath14** [comment]
I trained the stage 1 greyscale representation with nerf_pl and later trained stage 2 with colornerf. Now the structure is coming better. Thanks!
